.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           key: v1-deps-client-cache-{{ checksum "./client/yarn.lock" }}
       - run:
           name: Test client
-          command: yarn test:client
+          command: cd client && yarn && yarn test
```

Comment:
why are you doing the cd client when you have already set the working_directory?

Reply:
> why are you doing the cd client when you have already set the working_directory?
when i do not add the cd client i recieve the following build error
![Uploading Screenshot from 2019-09-19 21-37-19.png…]()